.changeset/unlucky-walls-drum.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+fix: set a no-op proxy config so that Vite HTTPS works
```

Comment:
```suggestion
fix: when using `@vitejs/plugin-basic-ssl`, set a no-op proxy config to downgrade from HTTP/2 to TLS since `undici` does not yet enable HTTP/2 by default
```